platform/testbed/src/integrationTest/resources/admin/snippet/create_get/01.lua

```diff
@@ -0,0 +1,21 @@
+sys = require('sys')
+
+err, create_snippet_req = sys.snippet.create({
+    name = 'test-snippet',
+    inputs = {},
+    value = [=[40 + 2]=]
```

Comment:
LoL what does [=[ do?

Reply:
[=[  ]=] escapes everything in between, remove it for consistency